### PR TITLE
draft for the data science service

### PR DESCRIPTION
**Pull request by adaortizcarbonell**
First draft for the data science service. Maybe particular examples of jobs that XAL has perfomed could be showcased here, or more uses of data science in the industry. Also someone needs to elaborate on the ML part.

### CODE REVIEW

File: content/services/datascience/index.md [hunk 1]
```diff
@@ -3,11 +3,38 @@ title: "Data Science and Machine Learning"
...
 tag: datascience
 ---
 
-Machine Learning is super cool. Artificial Intelligence is approaching
-SINGULARITY, and the human race is screaming with fear. Data Science used to be
-cool, now it's old school.
+Data Science is the science (and art) of extracting knowledge or insight from
+raw data, whether that is applied to research or to industry. Multiple
+disciplines are involved in the understanding of phenomena with data:
+mathematics, statistics, physics, computer science, for example.
```
Comment (skavhaug) [inline content/services/datascience/index.md]: We should iterate on the ingress to that it works better stand alone

**Comment by piaza** [comment]
I did not want to mess up the branch in case others are currently working on it. So here are a few suggestions from my side: 
- 1st paragraph, 1st sentence: that -> it
- 1st paragraph, 2nd sentence: Could be replaced by sth. like: "Extracting meaning from and interpreting data requires tools and methods of multiple disciplines such as mathematics, statistics, physics and computer science. 
- add project on optimisation of allocation of manpower relative to demand. We developed and implemented a full system from demand forecasting based on historical data to optimise work schedules. See my suggestion for implemented below
- 3rd paragraph: Other examples are optimisation problems where
production processes (and thus revenues) are maximized, maintenance operations are
minimized within safety regulations or work schedules are optimized in terms of the allocation of manpower relative to demand. 
- last paragraph: change understandable language --> comprehensive language because of repetition
- At SK, we have also been involved on a project related to lifecycle prediction models for equipment on wind farms. We are also currently running different models now which involve automated notifications for specific failure modes. We have lots of plots from the life monitoring of the machinery in the PP. Let me know if you would like to add some.

File: content/services/datascience/index.md (hunk 1 above)
Comment (skavhaug) [reply content/services/datascience/index.md]: Ok, so I did a minor error here, and will merge this with `servicespage` :)

**Comment by skavhaug** [comment]
@piaza Could you perhaps add your last point as a separate project under the projects-page?

Linking from these services to projects make a lot of sence (and perhaps vice versa)

**Pull request merged**